test/Test.Common.props

```diff
@@ -10,5 +10,10 @@
 
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsWindows Condition="'$(OS)' == 'Windows_NT'">true</IsWindows>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition=" '$(IsWindows)' != 'true' ">
+    <DefineConstants>$(DefineConstants);UNIX</DefineConstants>
```

Comment:
I don't `UNIX` gets used in your changes. Is this added for a general purpose?

Reply:
It was for the xUnit tests. They were unwanted.

Reply:
But it still makes sense to have it.

Reply:
@daxian-dbw 
This was used in the xUnit tests. It is not used anymore since we moved the tests to Pester. I am fine with either keeping it or removing it.